Project-II-Calculator/calculator/src/App.css

```
@@ -0,0 +1,14 @@
+.wrapper {
+  width: 100%;
+  height: 100%;
+  margin: 0 20%;
+  // border: 10px solid pink;
```

If you want to comment something out, it's better to highlight it and hit control + /, or command + / on a Mac. It will automatically use the correct comment format based on the filetype. 
```suggestion
  /* border: 10px solid pink; */
```